app-i18n/uchardet/uchardet-0.0.6.ebuild

```diff
@@ -7,7 +7,7 @@ inherit cmake-utils
 
 DESCRIPTION="An encoding detector library"
 HOMEPAGE="https://www.freedesktop.org/wiki/Software/uchardet/"
-SRC_URI="https://www.freedesktop.org/software/${PN}/releases/${P}.tar.xz"
+SRC_URI="https://www.freedesktop.org/software/uchardet/releases/${P}.tar.xz"
```

Comment:
https://github.com/gentoo/devmanual.gentoo.org/pull/68

It's basically got QA blessing already, and it's waiting for @gentoo/devmanual team to merge it.